pyproject.toml

```diff
@@ -92,3 +92,9 @@ module = [
   "traceloggingdynamic",
 ]
 ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+# The tests are not yet ready for --disallow-untyped-defs (or even --disallow-incomplete-defs)
```

Comment:
Oops, I forgot to run mypy on the tests. Let's use a less strict configuration for the tests, for now.